src/scss/_borders.scss

```diff
@@ -1,27 +1,25 @@
-////
-/// @group o-table
-/// @link http://registry.origami.ft.com/components/o-table
-////
-
-
 /// Horizontal borders
 @mixin oTableHorizontalBorders {
-	border-bottom: 1px solid _oTableGet('table-border-color');
-	border-top: 1px solid _oTableGet('table-border-color');
-
-	th:not([scope=row]) {
+	.o-table--horizontal-borders {
+		border-bottom: 1px solid _oTableGet('table-border-color');
 		border-top: 1px solid _oTableGet('table-border-color');
+
+		th:not([scope=row]) {
+			border-top: 1px solid _oTableGet('table-border-color');
+		}
 	}
 }
 
 /// Vertical borders
 @mixin oTableVerticalBorders {
-	border-left: 1px solid _oTableGet('table-border-color');
-	border-right: 1px solid _oTableGet('table-border-color');
-
-	// sass-lint:disable no-qualifying-elements
-	th[scope=row] {
+	.o-table--vertical-borders {
 		border-left: 1px solid _oTableGet('table-border-color');
+		border-right: 1px solid _oTableGet('table-border-color');
+
+		// sass-lint:disable no-qualifying-elements
+		th[scope=row] {
```

Comment:
if we don't enforce`[scope=row]`, we'll lose this styling, and its only in the markup, right?

Reply:
Correct, but we should enforce `scope=row`?

Reply:
👍